cinnabar/tests/test_measurements.py

```diff
@@ -13,6 +14,38 @@ def test_ground():
     assert g3 == g4
 
 
+@pytest.mark.parametrize('Ki,uncertainty,dG,dG_uncertainty,label,temp', [
+    [100 * unit.nanomolar, 10 * unit.nanomolar, -9.55 * unit.kilocalorie_per_mole,
+     0.059 * unit.kilocalorie_per_mole, 'lig', 298.15 * unit.kelvin],
+    [0.1 * unit.micromolar, 0.01 * unit.micromolar, -9.55 * unit.kilocalorie_per_mole,
+     0.059 * unit.kilocalorie_per_mole, 'lig', 298.15 * unit.kelvin],
+    [100 * unit.nanomolar, 10 * unit.nanomolar, -10.57 * unit.kilocalorie_per_mole,
+     0.066 * unit.kilocalorie_per_mole, 'lig', 330 * unit.kelvin],
+])
+def test_Ki_to_DG(Ki, uncertainty, dG, dG_uncertainty, label, temp):
+
+    Ki_to_DG = cinnabar.Measurement.from_experiment(Ki, label, uncertainty, '', temp)
+
```

Comment:
```suggestion

    assert Ki_to_DG.DG.units == unit.kilocalorie_per_mole
    assert Ki_to_DG.uncertainty.units == unit.kilocalorie_per_mole
```

A bit cautious, but because `pint` will do a conversion on equality, it's worth just double checking that the output units are what we expect them to be (particularly if we decide to jump to kj/mole later on, it'll make sure we have tests that check the unit return type).